### ISSUE TITLE
Need ability to run packaging tool from external media for Windows, Mac, and Linux.

### ISSUE THREAD

**Issue by tdilauro** [DataConservancy/dcs-packaging-tool#13]
#### Scenario / Use Case ####
- Preparation (per platform, per release)
    1. Install appropriate version on properly formated external drive.
- Routine use:
    1. Connect external drive to source (data) computer.
    2. Run Packaging Tool from external drive without installing on source computer
        - Save all artifacts of packaging on the external drive (no write access to source computer required)
    3. Disconnect external drive from source computer.
- Alternative use:
    - Save some or all packaging artifacts to the source computer (if desired and write access is availalble).

**Comment by htpvu** [comment]
Tim will DMS be providing us with the external drives? One for each platform? Who's in charge of these drives so that I know who to look for to get these when the time comes?

**Comment by tdilauro** [comment]
I don't think that there is any expectation that the PTG developers would be doing the installationto drives. The idea would be to provide enable the capability in the software (may already be there) and document how it should be installed on the drives; any limitations of using it that way; etc.

I will edit the UC to make this clearer.
